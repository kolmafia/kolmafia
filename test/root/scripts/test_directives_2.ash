script 'That_imported_script';

void main() {
	print( "Accessing the main function from another script." );
}
