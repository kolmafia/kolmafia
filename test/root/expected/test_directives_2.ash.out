Accessing the main function from another script.
